### DISCUSSION TITLE
Bug ? Maybe ?

### DISCUSSION THREAD

**Discussion by Elestralia** [Fluffernuttersandwich/Smorgasbord#5] [General]
Hello ! 

I'm new to this Modpack and i came acrros this:
![Sans titre](https://github.com/user-attachments/assets/7bbf89e3-ce39-401e-adc4-9166f54cac14)
I don't know if it's supposed to be in the game. In this save (creative on) i was able to farm zombies indefinitely and on the first save the zombie destroyed the cross stopping the spawn and my encumbered slot icon in my inventory was gone after that and i was able to have a full inventory without penalty. Is that supposed to behave like that ?

(Sry if my english is bad)

**Comment by Fluffernuttersandwich** [comment]
1. Your English is fine, no worries, I am able to understand what you are saying.
2. That Cross-shaped Tombstone is a zombie-spawning block, similar to a monster spawning block in Minecraft.  It will keep spawning zombies until it is destroyed.  This is intentional.  It's part of the CompoPack (CP) Mods.   
3. If you have the Creative and Debug Menus on, it will remove the encumbered slots on the inventory screen.  If you disable Creative and Debub menus, you will return to having encumbered slots.  It took me a little while to see that as well.

**Reply by Elestralia** [reply]
Make sense, i feel dumb for not paying attention it was the creative/debug. 
I was worried of the fact that it's easy to take advantage of the spawner but now i know and will use it more carefully.
Thanks you for the quick answer and for that great modpack, well made and really enjoyable.
